gwdetchar/lasso/plot.py

```diff
@@ -20,30 +20,24 @@
 """
 
 import os
+import atexit
 import shutil
 import tempfile
-import atexit
 import warnings
 
 from matplotlib import rcParams
 
 from gwpy.plot import Plot
 
-from ..plot import get_gwpy_tex_settings
-
 __author__ = 'Alex Urban <alexander.urban@ligo.org>'
 __credits__ = 'Alex Macedo, Jeff Bidler, Oli Patane, Marissa Walker, ' \
               'Josh Smith'
 
-# TeX settings
-tex_settings = get_gwpy_tex_settings()
-rcParams.update(tex_settings)
-
 
 # -- plotting utilities -------------------------------------------------------
 
 def configure_mpl():
```

Comment:
I think it makes sense to rename this to `configure_mpl_tex` or similar, since it _only_ impacts tex usage, not regular `mpl` usage.

Reply:
I agree, this is done in the latest commit